exe/wallet/Main.hs

```diff
@@ -130,6 +143,11 @@ Options:
   --bridge-port <INT>         port used for communicating with the http-bridge [default: 8080]
   --address-pool-gap <INT>    number of unused consecutive addresses to keep track of [default: 20]
   --size <INT>                number of mnemonic words to generate [default: 15]
+  --payment <PAYMENT>         address to send to and amount to send separated by @: '<amount>@<address>'
+
+Examples:
+  cardano-wallet transaction create --wallet-id 2512a00e9653fe49a44a5886202e24d77eeb998f --payment 22@2cWKMJemoBam7gg1y5K2aFDhAm5L8fVc96NfxgcGhdLMFTsToNAU9t5HVdBBQKy4iDswL # Create archive.tar from files foo and bar.
```

Comment:
At end of line:
```
# Create archive.tar from files foo and bar.
```

?

Reply:
ah - I was looking at how other CLIs format their help. Yes - let me fix that